### DISCUSSION TITLE
Make a new item for empty coilgun ammo boxes for easier single-player ammo management.

### DISCUSSION THREAD

**Discussion by OscillatorZ** [FakeFishGames/Barotrauma#8275] [Ideas and suggestions]
When I visit the shop in single-player all my coilgun ammo boxes get grouped together so I have no idea how many full ones I have or how many are empty. I have to go count them one by one, or I have to spend a really long tedious time walking back and forth in my sub deconstructing/recycling/throwing out empty coilgun ammo so that I can more easily assess how many to buy at the shop. It just seems like a chore thats more easily manageable in multiplayer when you can take advantage of downtime during navigation while the captain focuses on piloting. I would PAY the shopkeep to just take them off my hands en mass 🤣

**Comment by Danielkaas94** [comment]
My Barsuk is littered with boxes and old flares and glow sticks 🚮⚓

**Comment by ekqvist** [comment]
Thanks for the feedback. We're looking to address this in the future by indicating both the number of non-empty ammo boxes you own and the total number of ammo boxes you own on the store interface. This should be applied to a few other item types as well, such as fuel rods.
